### PR TITLE
Fix unitfile. Fix #65

### LINKED ISSUE #65: systemd file on Debian Buster not working
My Configuration:

varnish::addrepo: false
varnish::listen:
  - ':80
else are defaults.

The Systemd-Service file is generated with an "\" at the end of the Service section, this breaks the Startup of the varnish service with:
Oct 17 08:03:19 web01p varnishd[16430]: Error: Too many arguments ([Install]...)
Oct 17 08:03:19 web01p varnishd[16430]: (-? gives usage)

[varnish.service.txt](https://github.com/claranet/puppet-varnish/files/3738162/varnish.service.txt)

### PR DESCRIPTION
**Pull request by NITEMAN**
#### Pull Request (PR) description
As requested  on #68 by @fe80 , independent pull request addressing #65 

#### This Pull Request (PR) fixes the following issues
Fixes #65

### CODE REVIEW

**Comment by NITEMAN** [comment]
@fe80 I hope it's now ready to be merged

**Comment by NITEMAN** [comment]
@fe80 This is needed for #75 to work without issues

**Comment by NITEMAN** [comment]
Rebased on latest master

**Comment by NITEMAN** [comment]
Ping @fe80

**Comment by fe80** [comment]
Hello @NITEMAN 

You have a review for this pr :)

**Comment by NITEMAN** [comment]
Sorry @fe80 but I can't see it (maybe latest UX changes in GitHub made me lost my way), could you please provide me a link?

**Comment by fe80** [comment]
sorry it was not publish...

File: templates/varnish.service.erb [hunk 1]
```diff
@@ -42,10 +45,14 @@ ExecStart=/usr/sbin/varnishd <%= scope['::varnish::config::jail_opt'] %> \
   -p <%= k %>=<%= v %> \
 <% end -%>
   -S <%= scope['::varnish::secret_file'] %> \
-  -s <%= scope['::varnish::storage_type'] %>,<% if scope['::varnish::storage_type'] == 'file' -%><%= scope['::varnish::storage_file'] %>,<% end -%><%= scope['::varnish::storage_size'] %> \
-<% scope['::varnish::storage_additional'].each do |k| -%>
+  -s <%= scope['::varnish::storage_type'] %>,<% if scope['::varnish::storage_type'] == 'file' -%><%= scope['::varnish::storage_file'] %>,<% end -%><%= scope['::varnish::storage_size'] -%>
```
[the remark this answers is not in the document]
Comment (NITEMAN) [reply templates/varnish.service.erb]: Done with your suggestion (almost verbatim), just added a variable for readability

**Comment by NITEMAN** [comment]
@fe80 I hope it's ready now ;)

**Pull request merged**